app/main/views/organisations/index.py

```diff
@@ -442,6 +442,28 @@ def edit_organisation_can_approve_own_go_live_requests(org_id):
     )
 
 
+@main.route("/organisations/<uuid:org_id>/settings/edit-can-ask-to-join-a-service", methods=["GET", "POST"])
+@user_is_platform_admin
+def edit_organisation_can_ask_to_join_a_service(org_id):
+
+    form = YesNoSettingForm(
+        name="Can people ask to join services in this organisation?",
+        enabled="can_ask_to_join_a_service" in current_organisation.organisation_permissions,
```

Comment:
Since you’ve written the property on the model you might as well use it 😄 
```suggestion
        enabled=current_organisation.can_ask_to_join_a_service,
```

Reply:
True, I didn't have the property for a bit.